### DISCUSSION TITLE
Question regarding container update script and containers like n8n

### DISCUSSION THREAD

**Discussion by ajtatum** [tteck/Proxmox#1921] [General]
Hi,

I currently have n8n running in a VM using a Postgres DB and Redis Server hosted as their own containers. I've been thinking about installing the n8n container, but was curious if I install that using your script and then run your LXC update script, would that also update n8n to the most recent version without updating the environment variables that would point it to the Postgres and Redis servers (and other variables). Just asking at having n8n as a container and being able to update it with your script would definitely be more efficient than a dedicated VM as well as make the management of all my containers easier.

In that regard, does the update script update the OS and the specific app for that LXC created by your script? And what if the LXC was not created by your script, would it just basically do a apt update and apt upgrade?

Thanks for all your effort into making these scripts; they've been immensely valuable!

**Comment by tteck** [comment]
To update n8n LXC, you would type "update" in the LXC console. No environment variables change as it invokes `npm update -g n8n`
The Proxmox VE LXC Updater only updates the LXC OS.
